sequencer/src/block/namespace_payload/ns_payload.rs

```diff
@@ -0,0 +1,133 @@
+//! The only thing [`NsPayload2`] does is naively read from its payload given a
```

Comment:
This comment does not look up to date.

Reply:
fixed in https://github.com/EspressoSystems/espresso-sequencer/pull/1499/commits/5c9fe51fa9f95bb392d5c8446166068663ef0ac1